CHANGELOG.md

```diff
@@ -6,7 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [0.1.0]
-* Initial version
+
+- Initial version
```

Comment:
A pretty minor thing, but for our `cobalt-*` packages we order the CHANGELOG.md in reverse chronological order, so readers can see the latest change at the top of the file. Not sure if we want to do the same thing, I don't think it makes much of a difference in practice 🤷‍♂️

Reply:
Good idea - will do this on the next change